test/rules_pygen/rules_generator_test.py

```diff
@@ -0,0 +1,88 @@
+import operator
+import unittest
+import unittest.mock
+
+
+class WhenGeneratingBuildFilesTest(unittest.TestCase):
+    def test_that_wheel_compatibility_is_correct(self):
+        from rules_pygen.rules_generator import _check_compatibility
```

Comment:
We shouldn't be checking private functions in unit tests, all checks should happen in terms of the public interface that `rules_pygen` exposes, which in this case is `[...]Generator.run(...)`.